src/EA.Weee.Web/Areas/Admin/Views/CopyAatf/Copy.cshtml

```diff
@@ -0,0 +1,218 @@
+            @Html.Gds().TextBoxFor(m => m.ContactData.AddressData.TownOrCity, new { @class = "govuk-!-width-one-third", @maxlength = CommonMaxFieldLengths.TownCounty })
+        </div>
+
+        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.ContactData.AddressData.CountyOrRegion)">
+            @Html.Gds().LabelFor(m => m.ContactData.AddressData.CountyOrRegion)
+            @Html.Gds().ValidationMessageFor(m => m.ContactData.AddressData.CountyOrRegion)
+            @Html.Gds().TextBoxFor(m => m.ContactData.AddressData.CountyOrRegion, new { @class = "govuk-!-width-one-third", @maxlength = CommonMaxFieldLengths.TownCounty })
+        </div>
+
+        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.ContactData.AddressData.Postcode)">
+            @Html.Gds().LabelFor(m => m.ContactData.AddressData.Postcode)
+            @Html.Gds().ValidationMessageFor(m => m.ContactData.AddressData.Postcode)
+            @Html.Gds().TextBoxFor(m => m.ContactData.AddressData.Postcode, new { @class = "govuk-!-width-one-third", @maxlength = CommonMaxFieldLengths.Postcode })
+        </div>
+
+        <div class="govuk-form-group countries @Html.Gds().FormGroupClass(m => m.ContactData.AddressData.CountryId)" id="contact-country-dropdown-list">
+            @Html.Gds().LabelFor(m => m.ContactData.AddressData.CountryId)
+            @Html.Gds().ValidationMessageFor(m => m.ContactData.AddressData.CountryId)
+            @Html.DropDownListFor(m => m.ContactData.AddressData.CountryId, new SelectList(Model.ContactData.AddressData.Countries, "Id", "Name"), string.Empty, new { @class = "govuk-!-width-one-third govuk-input", autocomplete = "true", aria_labelledby = "contact-country-dropdown-list" })
+        </div>
+
+        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.ContactData.Telephone)">
+            @Html.Gds().LabelFor(m => m.ContactData.Telephone)
+            @Html.Gds().ValidationMessageFor(m => m.ContactData.Telephone)
+            @Html.Gds().TextBoxFor(m => m.ContactData.Telephone, new { @class = "govuk-!-width-one-third", @maxlength = CommonMaxFieldLengths.Telephone })
+        </div>
+
+        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.ContactData.Email)">
+            @Html.Gds().LabelFor(m => m.ContactData.Email)
+            @Html.Gds().ValidationMessageFor(m => m.ContactData.Email)
+            @Html.Gds().TextBoxFor(m => m.ContactData.Email, new { @class = "govuk-!-width-one-third", @maxlength = CommonMaxFieldLengths.EmailAddress })
+        </div>
+
+        <div class="govuk-!-padding-bottom-6"></div>
+        <button type="submit" name="save" class="govuk-button" id="site-details">
+            Save and continue
+        </button>
+        <div class="govuk-!-padding-bottom-6"></div>
+        <p>
+            <a href="@(Url.Action("Details", new { controller = "Aatf", area = "Admin", Id = Model.Id }))">Back</a>
```

Comment:
It is possible to put all the HTML that the create and copy share into a partial view?

Reply:
There are 3 views that have the same fields. have updated them to use partial view